odbc/tests/connection_tests.rs

```diff
@@ -13,7 +13,7 @@ fn test_invalid_connection() {
     );
 }
 #[allow(dead_code)]
-//#[test]
+#[test]
```

Comment:
I don't want to merge this until I get it working :)

It's working for me locally now (finally), but I think we are missing the necessary test infra to connect (we'll see in a second).

Locally, I'm connecting directly to a running mongod (since no real sql specific infra is necessary to check connecting).